tests/ci/data/profiles/rosa-hcp.yaml

```diff
@@ -62,6 +62,34 @@ profiles:
   account-role:
     path: "/test/hcp/"
     permission_boundary: "arn:aws:iam::aws:policy/AdministratorAccess"
+- as: rosa-hcp-ze
+  version: 4.14.38
```

Comment:
How about other versions? we use latest version usually

Reply:
Currently, zero_egress is only supported for 4.14.y versions. Once the GA is released, I can change this to latest